src/Bicep.Core/Syntax/SyntaxVisitor.cs

```diff
@@ -1,424 +1,139 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
+
+using Bicep.Core.Parsing;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Runtime.CompilerServices;
-using Bicep.Core.Parsing;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Bicep.Core.Syntax
 {
     public abstract class SyntaxVisitor : ISyntaxVisitor
```

Comment:
With the changes to the base class, the `ISyntaxVisitor` interface may not be needed anymore. I'll send a follow up PR to replace references to `ISyntaxVisitor` with `SyntaxVisitor`.